_posts/2021-12-01-commons-project-team.md

```diff
@@ -0,0 +1,34 @@
+
+- **[Jogebeth 'Joey' Salazar](https://about.me/gomezsalazar-jogebeth)** is a software engineer from Costa Rica, graduated in China, working in Internet Governance spaces in Europe and America. Continually advocating for free speech online and open source technologies, Joey focuses on policies, standards, and protocol implementations, in particular regarding the DNS and related privacy and censorship considerations. In the SDC project, Joey takes care of software development in OpenRefine's own codebase.
+
+- **[Sandra Fauconnier](https://meta.wikimedia.org/wiki/User:SFauconnier)** is an art historian who has engaged with digital projects in the cultural sector for many years. She is an active Wikimedian and a prolific user of OpenRefine, and has worked for the Wikimedia Foundation as part of the team which developed Structured Data on Wikimedia Commons. Sandra acts as product manager for the SDC project in OpenRefine.
+
+The team is mentored by Antonin Delpeuch.
+
+We are developing these new features in close collaboration with the Wikimedia community. You can read [monthly reports of ongoing work on meta.wikimedia.org](https://meta.wikimedia.org/wiki/Grants:Project/CS%26S/Structured_Data_on_Wikimedia_Commons_functionalities_in_OpenRefine/Timeline). Wikimedia-specific tasks are managed on Wikimedia's bug tracking system [Phabricator](https://phabricator.wikimedia.org/tag/openrefine/), and OpenRefine-specific development is [tracked on GitHub](https://github.com/OpenRefine/OpenRefine/projects/10). If you have a Wikimedia account, you can [subscribe](https://meta.wikimedia.org/wiki/Global_message_delivery/Targets/OpenRefine_and_SDC) to receive updates about the project on a talk page of your choice.
+
+# Structured Data on Wikimedia Commons?
+
+What is Structured Data on Wikimedia Commons? Quite a few OpenRefine users will be familiar with [Wikidata](https://www.wikidata.org/), the multilingual Linked Open Data knowledge base of the Wikimedia movement. Since 2019, media files (digital photographs, videos, sound files, etc) on Wikimedia Commons, [can also be described with multilingual linked data from Wikidata](https://commons.wikimedia.org/wiki/Commons:Structured_data).
+
+OpenRefine already includes support for Wikidata and, more recently, arbitrary Wikibases. The new Structured Data on Commons features in OpenRefine will build further upon this.
+
+Here's an early preview of structured data for a list of Wikimedia Commons files as displayed in OpenRefine:
+
+![Commons files with SDC and Wikitext in OpenRefine](https://openrefine.org/images/202111-commons-SDC-wikitext.png)
```

Comment:
You will need to include the image in your pull request too!

Reply:
Hmm, I was very confused when I thought about that, and already uploaded it to that specific place - so that's probably not needed? Or am I missing something?

See https://openrefine.org/images/202111-commons-SDC-wikitext.png - it's there 😛

Reply:
Ah ok! Ideally you should do this in the same pull request. One pull request can make changes to multiple files, and we generally try to group changes into consistent pull requests. If for some reason we did not merge this pull request (#94) then the repository would be left in an undesirable state (we would need to delete the file you uploaded).